pkg/kubelet/kubelet.go

```diff
@@ -3095,6 +3095,8 @@ func (kl *Kubelet) isContainerRuntimeVersionCompatible() error {
 		if result < 0 {
 			return fmt.Errorf("container runtime version is older than %s", dockertools.MinimumDockerAPIVersion)
 		}
+	case "rkt":
+		return kl.GetRuntime().(*rkt.Runtime).CheckVersion(rkt.MinimumRktBinVersion, rkt.RecommendedRktBinVersion, rkt.MinimumAppcVersion, rkt.MinimumRktApiVersion, rkt.MinimumSystemdVersion)
```

Comment:
Is it possible that the runtime is updated during kubelet running? If not, maybe we could just check at startup.